.gitmodules

```diff
@@ -4,8 +4,9 @@
   branch = develop
 [submodule "WW3"]
   path = WW3
-  url = https://github.com/NOAA-EMC/WW3
-  branch = dev/ufs-weather-model
+  #url = https://github.com/NOAA-EMC/WW3
+  #branch = dev/ufs-weather-model
+  url = https://github.com/JessicaMeixner-NOAA/WW3 
```

Comment:
@JessicaMeixner-NOAA Even though your develop branch is same as your sync-dev-to-ufs-20200308 (also the same as noaa emc ww3 develop branch as shown in your ww3 PR). But it looks to me your develop branch does not have the same history as the dev/ufs-weather-model, in other words your ww3 is not based on the top of the dev/ufs-weather-model branch, which makes the github version check fail. Is it intended to change dev/ufs-weather-model branch history?

Reply:
Unfortunately at some point the history of the dev/ufs-weather-model branch got messed up.  @aliabdolali and I  always make sure that the content of the dev/ufs-weather-model is what it should be but unfortunately so that the history of the ufs-weather-model stays there we can't delete the branch and start over with a clean history.  The WW3 develop branch still has a clean history though and at some point if we can make more frequent commits to ufs-weather-model with WW3 updates maybe we can move back to using the develop branch in the future.

Reply:
OK, I think the dev/ufs-weather-model history will be changed with your PR. If this is what you want to do, we will ignore the github version check error message.